app/javascript/stylesheets/ReviewMode.scss

```diff
@@ -3,21 +3,10 @@
 table {
   margin: auto;
   padding-top: 10px;
-  width: 800px;
 }
 
 tr, td {
   height: 50px;
-  padding: 20px 15px 20px 15px;
```

Comment:
This diff may be incorrect? I'll drop this change.

Reply:
@kevintli These are supposed to be kept, right?

Reply:
Yup that should still be in there